portafly/doc/ux_specs/global_components/modal.md

```diff
@@ -7,9 +7,10 @@
 
 ### Global modal specs
 * Modal dialogs are placed in the exact centre of the page.
-* Modal dialogs feature a "X" icon in the top right corner. Clicking the icon will dismiss the modal.
+* Modal dialogs feature an "X" icon in the top right corner. Clicking the icon will dismiss the modal.
 * Clicking on the foreground (everywhere on the page outside of the area covered by the modal), will dismiss the modal.
```

Comment:
This isn't supported by PF React. It should? https://www.patternfly.org/v4/documentation/react/components/modal